packages/sit-onyx/src/components/OnyxHeadline/types.ts

```diff
@@ -0,0 +1,12 @@
+export type OnyxHeadlineProps = {
```

Comment:
This is not possible because you can only export types from a .vue file but no values.
But we have `HEADLINE_TYPES` here which need to be exported so I decided to keep everything in one place in this extra file :)